packages/form-system/stories/01-overview.stories.mdx

```diff
@@ -0,0 +1,101 @@
+import { Story, Meta, html } from '@open-wc/demoing-storybook';
+import { render } from '@lion/core';
+import '@lion/input/lion-input.js';
+import { Validator } from '@lion/validate';
+import './helper-wc/h-output.js';
+
+<Meta title="Form Fundaments/Overview" />
+
+# Form System Overview
+
+This page should be used as a starting point when first using the Form System.
+It provides an overview of its essential building blocks and provides links to detailed explanations
+of most of its core concepts.
+
+## Building Blocks
+
+Our Form System is built from a set of very fundamental building blocks: `form control`s, `field`s and `fieldset`s.
+
+### Form Controls
+
+`Form control`s are the most fundamental building block of our Form System.
+They are the fundament of both `field`s, and `fieldset`s and provide a normalized, predictable api
+throughout the whole form.
+Every form element inherits from `FormControlMixin`.
+
+`FormControlMixin` creates the default html structure
+and accessibility is designed to be used in conjunction with the ValidateMixin and the FormatMixin.
```

Comment:
Lines 24 to 27 could be rewritten as:

```suggestion
Form elements inherit from `FormControlMixin`. This mixin is responsible with defining the accessible html structure. It should be used in conjunction with `ValidateMixin ` and `FormatMixin`.
```

Reply:
Language as chosen here was quite on purpose. They are designed to be used in conjunction with, but nothing is obliged. For instance, in the Fieldset we don't do this ourselves for historical reasons. We do however match the exact apis of those mixins in the Fieldset